bigbluebutton-html5/imports/ui/components/nav-bar/recordbutton/component.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component, PropTypes } from 'react';
+import styles from '../styles.scss';
+import NavBar from '../component';
+import Meetings from '/imports/api/meetings';
+
+export default class RecordButton extends Component {
+
+  render() {
+    let beingRecorded = Meetings.find().map(meeting => meeting.currentlyBeingRecorded);

```

Comment:
The component shouldn't use server functions (`find`). The `beingRecorded` state should be passed down as a property from the NavBar Container.